opensearchpy/connection/http_urllib3.py

```diff
@@ -115,8 +117,10 @@ def __init__(
         ssl_context: Any = None,
         http_compress: Any = None,
         opaque_id: Any = None,
+        metrics: Optional[Metrics] = None,
         **kwargs: Any
     ) -> None:
+        self.metrics = metrics
```

Comment:
Does it make sense to do `self.metrics = Metrics()` or `self.metrics = MetricsNone()` that doesn't do anything to avoid having to `if self.metrics` all over the place?

Reply:
Hello @dblock, 

We can't use `self.metrics = Metrics()` because Metrics is an abstract class. However, we can create a subclass called MetricsNone that sets all values to None for service_time, start_time, and end_time. Then, we'll set default metrics to MetricsNone(). Does this approach align with your thoughts?

Reply:
Yes. I think that would be better and future proof, no?